docker_mcp/server.py

```
@@ -135,23 +169,23 @@
     # Consolidated Tools Implementation
     
     async def docker_hosts(
```

Ruff (0.12.2)

`docker_hosts` is too complex (20 > 10)

(C901)